clients/cobol-lsp-vscode-extension/src/__tests__/commands/ClearCopybookCacheCommand.spec.ts

```diff
@@ -55,8 +52,14 @@ beforeAll(() => {
     vscode.window.showInformationMessage = jest.fn().mockImplementation((message: string) => Promise.resolve(message));
 });
 afterAll(() => {
+    console.log("wsPath", wsPath);
     if (fs.existsSync(wsPath)) {
```

Comment:
This is exactly what [rimraf](https://www.npmjs.com/package/rimraf) does.

Reply:
But do we really need this cleanup?

Reply:
Updated